diff --git a/cypress/e2e/tests/features/message-summary.spec.ts b/cypress/e2e/tests/features/message-summary.spec.ts
index 86fa5db..c9ff502 100644
--- a/cypress/e2e/tests/features/message-summary.spec.ts
+++ b/cypress/e2e/tests/features/message-summary.spec.ts
@@ -47,16 +47,19 @@ describe('Feature: message-summary', () => {
     chat.getMessage(2).isCompleted();
 
     cy.get('[data-testid="rancher-ai-ui-chat-message-box-1"] .inline-button').contains('See More').click();
+    cy.wait(500);
 
     cy.get('[data-testid="rancher-ai-ui-chat-message-box-1"] [data-testid="rancher-ai-ui-chat-message-formatted-content"]')
       .should('exist')
-      .and('be.visible');
+      .scrollIntoView()
+      .should('be.visible');
 
     cy.get('[data-testid="rancher-ai-ui-chat-message-box-1"] [data-testid="rancher-ai-ui-chat-message-formatted-content"]')
       .should('have.class', 'chat-msg-user-expanded');
 
     cy.get('[data-testid="rancher-ai-ui-chat-message-box-1"] .inline-button').should('contain.text', 'See Less');
 
+    chat.getMessage(1).scrollIntoView();
     chat.getMessage(1).containsText('Please analyse the Cluster');
 
     cy.wait(500);
